### PR TITLE
Fixes #RHIROS-1307 - Upgrade kruize image to 0.0.18_rm

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

**Comment by patilsuraj767** [comment]
We need - https://github.com/RedHatInsights/cicd-tools/pull/21 to fix the PR check.

File: internal/services/report_processor.go [hunk 1]
```diff
@@ -177,40 +153,73 @@ func ProcessReport(msg *kafka.Message) {
...
+	if len(d) > 0 {
+		notifications := d[0].Kubernetes_objects[0].Containers[0].Recommendations.Notifications
+		// 112101 is notification code for "Duration Based Recommendations Available".
+		if _, ok := notifications["112101"]; ok {
+			return true
+		} else {
+			return false
+		}
+	}
+	return false
```
Comment (saltgen) [inline internal/services/report_processor.go:224]: @patilsuraj767 Could we move this function to a utils file?

**Review by saltgen — CHANGES_REQUESTED** [review]

File: internal/types/kruizePayload/common.go [hunk 2]
```diff
@@ -87,13 +88,15 @@ type recommendedValues struct {
...
 	if metric, ok := data.(float64); ok {
 		return fmt.Sprintf("%.2f", metric)
-	} else {
-		return ""
 	}
+	if metric, ok := data.(int); ok {
+		return strconv.Itoa(metric)
+	}
+	return ""
+
```
[the remark this answers is not in the document]
Comment (patilsuraj767) [reply internal/types/kruizePayload/common.go:96]: Added comment on JIRA card.

**Review by saltgen — APPROVED** [review]
LGTM, thanks @patilsuraj767!

**Comment by patilsuraj767** [comment]
Thanks for the review @saltgen

**Pull request merged**